lib/board.rb

```diff
@@ -0,0 +1,74 @@
+class Board
+  attr_accessor :cells, :turn_count
+  #@cells = []
+def initialize
+  @cells = Array.new(9, " ")
+end
+
+def cells
+  @cells
+end
+
+def reset!
+  self.cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
Prefer `Array.new(9, " ")` as used above -- It's more concise and will prevent errors from repeatedly typing `" "`.